### PR TITLE
Remove Twitter; use Mastodon

### PR DESCRIPTION
**Pull request by dr0i**
See #364.

- update fontawesome to 6.4.2 (gains mastodon) by copy new css to old css
- refactor "fa" (fontawesome) to "fa-brands" for brands (github and twitter)

### CODE REVIEW

**Comment by dr0i** [comment]
Deployed to stage, see http://stage.lobid.org/gnd @acka47 for functional review.

**Comment by dr0i** [comment]
Wait , this is not ready - the Blog (pencil) icon is missing.

**Review by acka47 — APPROVED** [review]
+1

**Comment by dr0i** [comment]
Assigning @sroertgen for code review.

File: app/views/main.scala.html [hunk 1]
```diff
@@ -135,8 +135,8 @@
 	                  <a href="http://lobid.org/warranty">Gewährleistung</a> | 
                     <a href="http://www.hbz-nrw.de/impressum">Impressum</a> | 
                     <a href="https://github.com/hbz/lobid/blob/master/conf/Datenschutzerklaerung_lobid.textile">Datenschutz</a> | 
-                    <a href="http://twitter.com/lobidorg"><i class="fa fa-twitter" aria-hidden="true"></i> Twitter</a>&nbsp; 
-                    <a href="http://github.com/hbz/lobid-gnd"><i class="fa fa-github" aria-hidden="true"></i> GitHub</a>&nbsp; 
+                    <a href="https://openbiblio.social/@@lobid"><i class="fa-brands fa-mastodon" aria-hidden="true"></i> Mastodon</a>&nbsp;
```
Comment (sroertgen) [inline app/views/main.scala.html:138]: Is the double `@@` correct?

Comment (dr0i) [reply app/views/main.scala.html:138]: Yes, it's the way to escape something in scala.

**Comment by dr0i** [comment]
Deployed to https://lobid.org/gnd - closed.

**Pull request merged**